modules/core/src/lib/constants.js

```diff
@@ -26,19 +26,20 @@
 export const COORDINATE_SYSTEM = {
   // Positions are interpreted as [lng, lat, elevation]
   // lng lat are degrees, elevation is meters. distances as meters.
-  LNGLAT: 1.0,
-  LNGLAT_EXPERIMENTAL: 4.0,
+  LNGLAT: 1,
+  LNGLAT_EXPERIMENTAL: 1,
+  LNGLAT_DEPRECATED: 5,
 
   // Positions are interpreted as meter offsets, distances as meters
   // Planned to deprecate in later versions
```

Comment:
Move this comment one line down. Absolutely no plans to deprecated `METER_OFFSETS`

Reply:
Done.